### ISSUE TITLE
[From Hugging Face] Can we help in any way? 

### ISSUE THREAD

**Issue by patrickvonplaten** [lucidrains/muse-maskgit-pytorch#14]
Hey @lucidrains,

Impressive that you already started a MUSE PyTorch implementation here! At Hugging Face, we also started thinking/designing a MUSE reproduction.

We are quite excited about on open-reproduction effort where the outcome is a checkpoint that is competitive to MUSE available for everybody. The advantage of MUSE in our opinion to existing models is:

- Should be faster than Stable Diffusion, Imagen & co.
- Inpainting works out of the box
- No need for complex schedulers

Can we help you in any way? Starting to curate a dataset etc...? 

cc @patil-suraj

**Comment by lucidrains** [comment]
@patrickvonplaten i'm afraid i won't have any time to embark on a multi month training run, as i have my eyes set on open sourcing some protein diffusion + text to 3d. do you have any solid research engineers that is willing to set sail? i am willing to put in the engineering refinements and unblock you on the code end

**Comment by lucidrains** [comment]
@patrickvonplaten i'm afraid i won't have any time to embark on a multi month training run, as i have my eyes set on open sourcing some recent protein diffusion work + text to 3d. do you have any solid research engineers / scientists that is willing to set sail? i am willing to put in the engineering refinements and unblock you on the coding end

**Comment by lucidrains** [comment]
it would seem as though we are slowly removing code, data, and even compute. and yet now we are realizing trainers with the appropriate background knowledge to carry out training the neural network is still in short supply, despite our best efforts to educate. hopefully this changes..

**Comment by lucidrains** [comment]
@patrickvonplaten is it possible to galvanize someone from the huggingface community to lead training? i was under the impression you had a large community of trainers from Bloom

**Comment by lucidrains** [comment]
>  Starting to curate a dataset etc...?

realistically, you'd use a filtered subset of laion?

**Comment by patrickvonplaten** [comment]
Think we don't (yet) have anybody willing to spend most of their time on it, but we're actively discussing the idea - will keep you posted! 

Yeah exactly we were thinking filtered subset of laion or https://huggingface.co/datasets/kakaobrain/coyo-700m

**Comment by lucidrains** [comment]
@patrickvonplaten i see, have you tried talking to Boris Dayma? he's very good

otherwise if time is not an issue, how about the thousands of people who participated for Bloom? may be good to form an initiative there

**Comment by patrickvonplaten** [comment]
@borisdayma let's interested in a MUSE-mini project ;-)

**Comment by patrickvonplaten** [comment]
@borisdayma let's interested in a MUSE-mini project ;-)

**Comment by nightscape** [comment]
In case anybody picks this up, it would be super awesome to record it or stream it on Twitch!

**Comment by bzantium** [comment]
@patrickvonplaten Is there any progress or open discussion channel for reproducing MUSE-mini?

**Comment by patrickvonplaten** [comment]
Cool, also cc @patil-suraj we're still discussed if we want to give this one a try

**Comment by bzantium** [comment]
@patrickvonplaten @patil-suraj Could you let me know if there exists any progress on this discussion? :)

**Comment by ZeroCool940711** [comment]
I think one way Huggingface can help is by adding support for training and inference with Muse on diffusers, diffusers already comes with a lot of performance and memory optimizations and is pretty much the standard for a lot of things so, having Muse support in there would make things a lot easier for everyone.

**Comment by patrickvonplaten** [comment]
We're slowly getting started on an open-reproduction effort, with a small core team. Will soon announce more about it :-)

**Comment by nightscape** [comment]
@patrickvonplaten not sure if this would be a hindrance, but recordings of some sessions would be super-interesting and helpful for people like me who want to get deeper into the practical side of DL.
